ui/cypress/e2e/README.md

```diff
@@ -0,0 +1,31 @@
+
+Depends on host:
+cy.onlyOn('localhost')
+
+onlyOn('mac', () => {
```

Comment:
Argh, was looking for this earlier in the docs, couldn't find it. I'll add a proper helper README as follow-up work. Help needed.